SmartDeviceLink/SDLRegisterAppInterface.m

```diff
@@ -103,6 +125,29 @@ - (instancetype)initWithAppName:(NSString *)appName appId:(NSString *)appId lang
     return self;
 }
 
+#pragma mark Initializer Helpers
+
+- (instancetype)initWithAppName:(NSString *)appName languageDesired:(SDLLanguage)languageDesired {
```

Comment:
This doesn't have a public corollary. It doesn't need to exist. All of the other initializers should be able to call the one initializer with all properties I think. All of these initializer implementations need some cleanup.

Reply:
Removed